### DISCUSSION TITLE
Contribute for beginners

### DISCUSSION THREAD

**Discussion by MicheleVerriello** [carbon-language/carbon-lang#1774] [Getting started]
Hi guys, I'd like to help to the development and the creation of new feature for carbon. I know Java very well but I'm new to the open source world, and I never developed a programming language. Do you think there is something to do for me (easy tasks, fix something)? Thanks!

**Comment by MrKrishnaAgarwal** [comment]
you can look at the contributing.md and issues for contributing.

Thanks

**Comment by geoffromer** [comment]
You could also take a look at the ["good first issue"](https://github.com/carbon-language/carbon-lang/labels/good%20first%20issue) label. We're trying to apply that to issues that could make a good starter project.
